src/components/PageComponents/Notification/NotificationItem/NotificationItem.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { Box, Chip, IconButton, Typography } from '@mui/material';
+import styles from './NotificationItem.styles';
+import PropTypes from 'prop-types';
+import Sms from '@mui/icons-material/SmsOutlined';
+import InfoOutlined from '@mui/icons-material/InfoOutlined';
+import ErrorRounded from '@mui/icons-material/ErrorRounded';
+import Close from '@mui/icons-material/Close';
+import { useSelector } from 'react-redux';
+import { selectCurrentUser } from '../../../../redux/auth/authSlice';
+import { useDeleteNotificationMutation, useMarkAsReadMutation } from '../../../../redux/services/notificationsApiSlice';
+import TimeAgo from '../../../UI/TimeAgo';
+import { useTranslation } from 'react-i18next';
+
+const iconMap = {
+  INTERVIEW_REQUEST_EXPIRED: <Sms />,
+  INTERVIEW_REJECTED: <InfoOutlined />,
+  INTERVIEW_SCHEDULED: <ErrorRounded />,
+  GREETING: <ErrorRounded />,
+};
+
+const NotificationItem = ({ data }) => {
+  const { t } = useTranslation();
+  
+  const { id, type, read, createAt, payload } = data;
+  const payloadObj = JSON.parse(payload);
+  const {role,scheduledDateTime} = payloadObj;
+  
+  const currentUser = useSelector(selectCurrentUser);
```

Comment:
its a minor thing, but just due to consistence of code
const { data: currentUser } = useSelector(selectCurrentUser);